pkgs/by-name/sm/smoked-salmon/package.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  cambia,
+  ffmpeg-headless,
+  flac,
+  lame,
+  mp3val,
+  oxipng,
+  python3Packages,
+  sox,
+}:
+let
+  runtimeDeps = [
+    cambia
+    oxipng
+    ffmpeg-headless
+    flac
+    lame
+    mp3val
+    sox
+  ];
+in
+python3Packages.buildPythonApplication rec {
+  pname = "smoked-salmon";
+  version = "0.9.5-unstable-2025-06-15";
```

Comment:
The 0.9.5 release has a couple of bugs/typos, but there hasn't been a release with the bug fixes yet.

Will update once they release `0.9.6`.